tests/test_models/test_engine/test_db_storage.py

```diff
@@ -86,3 +86,37 @@ def test_new(self):
     @unittest.skipIf(models.storage_t != 'db', "not testing db storage")
     def test_save(self):
         """Test that save properly saves objects to file.json"""
+
+
+class TestDBCount(unittest.TestCase):
+    """ Test for count func """
+
+    def test_count_int(self):
+        """Test that return type is int"""
+        self.assertIs(type(models.storage.count()), int)
+
+    def test_count_class_int(self):
+        """Test that return type is int"""
+        for klass in classes:
+            self.assertIs(type(models.storage.count(klass)), int)
+
+    def test_correct_count_return(self):
+        """ Test the return value of count """
+        self.assertEqual(models.storage.count(), len(models.storage.all()))
+
+    def test_correct_count_class_return(self):
+        """ Test the return value of count when class is passed """
+        for klass in classes:
```

Comment:
Camilo this name is not meaningful enough

Reply:
Changed to 'class_in_dic'